metricbeat/magefile.go

```diff
@@ -96,6 +98,27 @@ func Fields() error {
 	return mage.GenerateFieldsYAML("module")
 }
 
+func ExportDashboard() error {
+	module := os.Getenv("MODULE")
```

Comment:
Do all of this work from the shared code in dev-tools/mage/dashboard.go so that you don't have to duplicate it in every magefile.go. Hopefully in the future there will be an [import mechanism](https://github.com/magefile/mage/issues/108) in mage to avoid duplication in an even better way.